prac_05/wimbledon.py

```diff
@@ -0,0 +1,51 @@
+"""
+Word Occurrences
```

Comment:
Was this unintentional? (Don't worry, I've left text in docstrings that was supposed to be removed before!)